### DISCUSSION TITLE
Release v0.21.2 Beta

### DISCUSSION THREAD

**Discussion by PrestonN** [FreeTubeApp/FreeTube#5469] [Announcements]
## Summary

This is a hotfix to fix the issues with the Local API unable to play videos. Small update today but we'll have more to come later down the line.

## Fixes

* Update YouTube.js to 10.2.0 to fix latest Local API issues by @absidue in https://github.com/FreeTubeApp/FreeTube/pull/5466

<hr /><em>This discussion was created from the release <a href='https://github.com/FreeTubeApp/FreeTube/releases/tag/v0.21.2-beta'>Release v0.21.2 Beta</a>.</em>

**Comment by Draluy** [comment]
Thank you!

**Comment by b1k3rdude** [comment]
Thanks for working on this, every 3rd party YT player is now been activly targeted by Alphabet/Google, so projects like this are very much appreciated.

**Reply by b1k3rdude** [reply]
So YT are at it again, videos from any source wont play now :-/

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Follow thread for updates: https://github.com/FreeTubeApp/FreeTube/issues/5502#issuecomment-2261544217
